packages/xstate-react/test/useActor.test.tsx

```diff
@@ -198,7 +214,9 @@ describe('useActor', () => {
         done();
       }
 
-      return <ChildTest actor={state.context.actorRef} />;
+      const { actorRef } = state.context;
+
+      return actorRef ? <ChildTest actor={actorRef} /> : null;
```

Comment:
This is correct from the perspective of the types but it stops fully validating what this test is about. The test's title is:
> initial spawned actor should be immediately available

```suggestion
      return <ChildTest actor={actorRef!} />;
```